includes/misc-functions.php

```diff
@@ -1050,3 +1050,35 @@ function give_is_terms_enabled( $form_id ) {
 		return false;
 	}
 }
+
+
+/**
+ * Delete donation stats cache.
+ * @todo Resolve stats cache key naming issue. Currently it is difficult to regenerate cache key.
+ *
+ * @since 1.8.7
+ *
+ * @param string|array $date_range Date for stats.
+ *                           Date value should be in today, yesterday, this_week, last_week, this_month, last_month, this_quarter, last_quarter, this_year, last_year.
+ *                           For date value other, all cache will be removed.
+ *
+ * @param array  $args
+ *
+ * @return WP_Error|bool
+ */
+function give_delete_donation_stats( $date_range = '', $args = array() ) {
```

Comment:
@DevinWalker Currently, it is very difficult to get a transient key for any stat cache. That why flush all cache will be good. After refactoring reports, we can update this function too.